config.go

```diff
@@ -2,35 +2,57 @@ package main
 
 import (
 	"flag"
+	"os"
 
 	"github.com/vharitonsky/iniflags"
 )
 
 var (
+	VERSION    = "1.6.0"
```

Comment:
I think this is up to the maintainer to change when a new version is tagged.

Reply:
This was my bad - this constant got lost in the merge so I restored it and just bumped the version for no good reason